sdk/samples/iot/templates/iothub/test-resources-post.ps1

```diff
@@ -56,6 +37,20 @@ $fingerprint = Get-Content -Path .\fingerprint.txt
 # Get the hub as an object
 $hub_obj = Get-AzIotHub -ResourceGroupName $ResourceGroupName -Name $iothubName
 
+$retryCount = 1
+while ($retryCount -lt 5 -and $hub_obj.Properties.State -ne "Active")
+{
+  Write-Host "AzIotHub is not yet active so sleeping for $retryCount seconds"
+  Start-Sleep -Seconds $retryCount
```

Comment:
Nice comment and logic. Fun linear back-off time for the retry (1s, then 2s, ...). Is 10 second good enough as the max timeout?

Reply:
I didn't want to always blindly sleep for a fixed amount of time. I wanted to try and continue as quickly as possible.